### PR TITLE
1504593 Refactor health check playbooks

### PR DESCRIPTION
**Pull request by mtnbikenc**
- Standardize play/tasks naming
- Move install checks to separate playbook with checkpointing
- Correct 'docker_storage' tags

https://bugzilla.redhat.com/show_bug.cgi?id=1504593

### CODE REVIEW

**Comment by mtnbikenc** [comment]
It seems like we should be using the `@preflight` group for the checks during an install.  The only difference between what we have listed and what is in `@preflight` is `package_update`.

File: playbooks/common/openshift-checks/install.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  tasks:
+  - name: Set Health Check 'In Progress'
+    set_stats:
+      data:
+        installer_phase_health: "In Progress"
+      aggregate: false
+
+- name: OpenShift Health Checks
+  hosts: oo_all_hosts
+  any_errors_fatal: true
```
Comment (mtnbikenc) [inline playbooks/common/openshift-checks/install.yml]: I'm also adding `any_errors_fatal: true` here like I did for #5741

File: roles/openshift_health_checker/openshift_checks/docker_storage.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ class DockerStorage(DockerHostMixin, OpenShiftCheck):
     """
 
     name = "docker_storage"
-    tags = ["pre-install", "health", "preflight"]
+    tags = ["health", "preflight"]
```
Comment (mtnbikenc) [inline roles/openshift_health_checker/openshift_checks/docker_storage.py]: Before this change the groups were defined as:
```
@etcd = etcd_imagedata_size,etcd_traffic,etcd_volume
@health = curator,diagnostics,docker_storage,elasticsearch,etcd_traffic,etcd_volume,fluentd,fluentd_config,kibana,logging_index_time,ovs_version
@logging = curator,elasticsearch,fluentd,kibana,logging_index_time
@pre-install = docker_storage
@preflight = disk_availability,docker_image_availability,docker_storage,memory_availability,package_availability,package_update,package_version
```

File: playbooks/common/openshift-checks/install.yml (hunk 1 above)
Comment (sosiouxme) [reply playbooks/common/openshift-checks/install.yml]: Interesting, were any errors not fatal before?

Comment (mtnbikenc) [reply playbooks/common/openshift-checks/install.yml]: I confirmed this was a problem when troubleshooting for https://github.com/openshift/openshift-ansible/pull/5741

**Comment by mtnbikenc** [comment]
[ERROR] wait_for_fluentd_to_catch_up
/retest

**Comment by mtnbikenc** [comment]
[ERROR] wait_for_fluentd_to_catch_up
https://github.com/openshift/origin/issues/16005
/retest

**Comment by sdodson** [comment]
If logging is the only failing tests ping me for a manual merge.

**Comment by mtnbikenc** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1504593
/kind bug
/restest

**Comment by mtnbikenc** [comment]
Test statuses on this PR are stale.  logging and system-containers show as success in details. upgrade shows as failed in details.  Unable to get tests to restart.

**Comment by mtnbikenc** [comment]
@kargakis Something still seems broken with the tests on this PR.  Thoughts?

**Pull request merged**